### PR TITLE
added x-ref from scale reference to replicas in v3 Compose

### PR DESCRIPTION
**Pull request by londoncalling**
### What changed

- Added x-ref to `replicas` from `scale` in Compose reference per user feedback

### Related issues

#1816 

### Reviewers

@funkydorian, @shin- 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by shin-** [comment]
cc @dnephin 

I guess it's similar in effect but not exactly equivalent.

File: compose/reference/scale.md [hunk 1]
```diff
@@ -13,3 +13,10 @@ Sets the number of containers to run for a service.
 Numbers are specified as arguments in the form `service=num`. For example:
 
     docker-compose scale web=2 worker=3
+
+>**Tip:** Alternatively, in
+[Compose file version 3.x](/compose/compose-file/index.md)
+with Docker Engine 1.13 and above, you can specify
```
Comment (shin-) [inline compose/reference/scale.md]: "Engine 1.13" is redundant since we're already talking about "Compose file version 3.x"

File: compose/reference/scale.md [hunk 2]
```diff
@@ -13,3 +13,10 @@ Sets the number of containers to run for a service.
...
+[`replicas`](/compose/compose-file/index.md#replicas) under
+[`deploy`](/compose/compose-file/index.md#deploy) as part of the service
```
Comment (shin-) [inline compose/reference/scale.md]: I'll leave it up to you, but is "under deploy" here really necessary? It should become obvious as soon as readers click through the `replicas` link.

File: compose/reference/scale.md [hunk 3]
```diff
@@ -13,3 +13,10 @@ Sets the number of containers to run for a service.
...
+configuration for a [swarm](/engine/swarm/).
```
Comment (shin-) [inline compose/reference/scale.md]: "for Swarm mode"

**Comment by londoncalling** [comment]
@shin- , thanks, updated per your comments. 

Per your suggestion, I tried leaving out the additional reference to "under `deploy`" but the way the examples are shown in the Compose file, seems more thorough with it in. It might be a little overkill, but I don't think it hurts anything, and possibly adds clarity.

I'll wait and see if @dnephin weighs in now.

**Comment by londoncalling** [comment]
@shin- @dnephin @mstanleyjones leveraged (and fixed a small issue) in #1842 which enabled us to surface H1 topic titles in the right side "on-page" navigation menus if needed. I think this is good for these Compose file format references as an extra reminder of which version you are looking at.

Merging this as soon as it builds and passes checks to get these fixes in (especially fix re: #1842  because it has global impact on docs).

@dnephin feel free to suggest any changes to the tip on `scale.md` if you don't like it.

File: _layouts/docs.html [hunk 4]
```diff
@@ -301,7 +301,6 @@
 								{% endif %}
 								<div id="side-toc"><div id="side-toc-title">On this page:</div><div id="side-toc-contents">{% include toc_pure_liquid.html html=my_content sanitize=true class="inline_toc" id="my_toc" toc_min=my_min toc_max=my_max page_name=my_name %}</div></div>
 								{% endunless %}
-								<div>{{content}}</div>
```
Comment (dnephin) [inline _layouts/docs.html]: This seems unrelated. I guess it was just a cleanup?

File: compose/reference/scale.md [hunk 5]
```diff
@@ -13,3 +13,9 @@ Sets the number of containers to run for a service.
 Numbers are specified as arguments in the form `service=num`. For example:
 
     docker-compose scale web=2 worker=3
+
+>**Tip:** Alternatively, in
+[Compose file version 3.x](/compose/compose-file/index.md), you can specify
+[`replicas`](/compose/compose-file/index.md#replicas)
+under [`deploy`](/compose/compose-file/index.md#deploy) as part of the
+service configuration for [Swarm mode](/engine/swarm/).
```
Comment (dnephin) [inline compose/reference/scale.md]: Do you think it's worth including the caveat that `replicas` only works for `docker stack deploy` and not `docker-compose up`?

Comment (funkydorian) [inline compose/reference/scale.md]: i think this should be indicated as today one of developer over here got mixed up because of that
